### DISCUSSION TITLE
Meeting Notes: Metapunks present their new project - Cometa

### DISCUSSION THREAD

**Discussion by AlgoRhythMatic** [asastats/channel#27] [Meeting]
### Meeting Notes: Metapunks present their new project - Cometa

Meeting Attendees:
Babbexx22 and AlgoRhythMatic met w/ Florian Launay (communications/marketing), and Daria Yakovleva (development)

Landing page for Metapunks upcoming "Cometa" DeFi/Yield Farming site: 
https://cometa.farm/

Cometa Whitepaper: 
https://metalabs.gitbook.io/cometa/

Initial points of presentation by Metapunks:
- Metapunks is expanding from their original NFT offerings into the DeFi space, and calling the project "Cometa"
- They will be launching their yield-farming platform before summer, which will feature:
    - Metatoken/ASA LP staking -> rewarded in Metatokens
    - Stake metatoken -> rewarded in ASA coins (similar to Yieldly)
    - liquidity as a service (LaaS) - the use of autocompounding instead of swap fees
    - Reach-based smart contracts w/ autocompounding
    - Easy to use UI w/ selectable automated "auto-strategies"

Questions from ASA Stats:
- Q: How does Metapunks' NFT platform relate to their yield farming (Cometa) platform?
    - A: there will be added benefits for Metapunks NFT holders such as DAO membership, extra airdrops, extra dashboards/functionalities etc.
- Q: What types of yield-farming options will be available to ASA Stats?
    - A: metatoken/asastats LP -> rewarded in metatoken, stake meta -> get asastats
- Q: Are there minimum amounts needed in terms of TVL for pools?
    - A: No minimum amount...TBD by ASA Stats. Note: we did not discuss cost related to their staking services.

API partnership possibilities discussed:
- Metapunks/cometa are interested in the potential to use ASA Stats statistics embedded in their dashboard.  We discussed Metapunks participation in API MVP discussions, and will provide links to the Github discussion here: https://github.com/asastats/channel/discussions/1
- We noted that ASA Stats would be interested in Metapunks Yield Farming API (once it is ready) as well.
- We discussed the topic of DAO and on-chain voting vs. other methods.  We invited Metapunks to participate on our ongoing discussions on this topic here: https://github.com/asastats/dao-discussion/discussions/10 

Summary: a very interesting conversation w/ Florian and Daria from Metapunks team.  They'd like for ASA Stats to participate in their Yield Farming at launch, and would like to understand our interest.  As well, they are interested in some API partnership, w/ potential to provide data in both directions.  We invited the Metapunks team to participate in discussions related to API MVP as well as DAO voting, and will stay in touch over Twitter.

**Comment by ipaleka** [comment]
Great news, we'd be happy to discuss our participation in their staking programs once the audits are done.

Great move to invite them to those two discussions, especially the MVP discussion as we can maybe even focus in the early phase on the parts they need. But also the DAO discussion, as we welcome every Algorand dApp provider with DAO in their minds. :)
